### PR TITLE
Newseed: Adjustement to seed2 PR

### PR DESCRIPTION
**Pull request by Peartes**

### CODE REVIEW

File: src/footer/footer.html [hunk 1]
```diff
@@ -29,33 +29,33 @@
...
-          <a href="#" target="_blank" rel="noopener noreferrer"
-            >Upcoming Launches</a
-          >
-          <a href="#" target="_blank" rel="noopener noreferrer"
-            >Host a Launch</a
-          >
-          <a href="#" target="_blank" rel="noopener noreferrer"
-            >Prime Launch Docs</a
-          >
+          <a href="#" target="_blank" rel="noopener noreferrer">
```
Comment (dkent600) [inline src/footer/footer.html:32]: really these anchors should be on one line.  that is how they were.  as you have it, one is introducing whitespace, with potential consequences (that I don't know how to fully predict).  further I think, within reason, one-liners are cleaner in appearance.  Same for all these similar changes across this file.

File: src/newSeed/stage1.ts [hunk 2]
```diff
@@ -1,21 +1,17 @@
-import { RouteConfig } from "aurelia-router";
 import { BaseStage } from "./baseStage";
 // Let's import the utils helper
 import {Utils} from "../services/utils";
 export class Stage1 extends BaseStage {
-  activate(_params: unknown, routeConfig: RouteConfig): void {
```
Comment (dkent600) [inline src/newSeed/stage1.ts:6]: good to do this, needs to happen in stage2.ts as well.

**Review by dkent600 — CHANGES_REQUESTED** [review]
Still need:

https://github.com/PrimeDAO/prime-launch-dapp/pull/9#discussion_r616865328

File: src/footer/footer.html (hunk 1 above)
Comment (Peartes) [reply src/footer/footer.html:32]: Prettier. I'll increase print width

File: .vscode/settings.json [hunk 3]
```diff
@@ -6,5 +6,7 @@
   "[typescript]": {
     "editor.formatOnSave": false
   },
-  "files.eol": "\n"
+  "files.eol": "\n",
+  "prettier.printWidth": 180,
```
Comment (dkent600) [inline .vscode/settings.json:10]: I don't think you should use prettier, and you shouldn't set a rule to limit line lengths.  Much better to just rely on vscode and (es/te)lint.

It is your choice...but I really don't want to deal in PRs with prettier reformating...

**Review by dkent600 — CHANGES_REQUESTED** [review]

**Review by dkent600 — APPROVED** [review]

**Pull request merged**